tools/validators/instance_validator/tests/fake_instances/GOOD/states_case_sensitive.yaml

```diff
@@ -20,7 +20,7 @@ US-SEA-BLDG1-GUID:
   code: US-SEA-BLDG1
 
 FC2-1-1-GUID:
-  cloud_device_id: 'abc123'
+  cloud_device_id: '2619178366980754'
```

Comment:
change cloud_device_id to something more generic like 1234567890123456

Reply:
Done.